tests/tests_e3/anod/spec_test.py

```diff
@@ -147,3 +147,12 @@ class GeneratorDisabled(Anod):
 
     assert spec_enable.args == {"q1": True}
     assert spec_disable.args == {"q1": ""}
+
+
+def test_missing_property():
+    class NoProperty(Anod):
+        def source_pkg_build(self) -> list:
+            return []
+
+    noproperty = NoProperty(qualifier="", kind="source")
+    assert noproperty.source_pkg_build == []
```

Comment:
We may want a bit more here. I know that I sometimes forgot the `@property` for `build_deps` (for instance), and calling the `anod.build()` failed. Would it be a good test ?

Reply:
we don't have a full anod currently in e3-core though

Reply:
Oh right ... sorry :)